run/john.zsh_completion

```diff
@@ -48,19 +48,19 @@ _arguments -C -s \
   '(--no-mask)'--no-mask'[used with --test for alternate benchmark w/o mask]' \
   '(--skip-self-tests)'--skip-self-tests'[skip self tests]' \
   '(--test-full)'--test-full'[run more thorough self-tests]' \
-  '(--users'--users='[do not load these users only]:user login or uid' \
-  '(--groups'--groups='[do not load these group only]:group gid' \
-  '(--salts'--salts='[load salts with(out) COUNT (to MAX) hashes]:salts' \
-  '(--costs'--costs='[load salts with(out) cost value Cn (to Mn)]:costs' \
+  '(--users)'--users=-'[do not load these users only]:user login or uid' \
+  '(--groups)'--groups=-'[do not load these group only]:group gid' \
+  '(--salts)'--salts=-'[load salts with(out) COUNT (to MAX) hashes]:salts' \
+  '(--costs)'--costs=-'[load salts with(out) cost value Cn (to Mn)]:costs' \
   '(--make-charset)'--make-charset=-'[make a charset file. It will be overwritten]:charset file:_files' \
   '(--pot)'--pot=-'[pot file to use]:pot file:_files' \
-  '(--node)'--node=-'[this node's number range out of TOTAL count]:number' \
+  '(--node)'--node=-'[this nodes number range out of TOTAL count]:number' \
```

Comment:
If we cannot have an apostrophe in there (but perhaps we can with some kind of escaping?), then maybe re-word to "number range of this node out of TOTAL count".

Reply:
" ... ' ... " does the trick

Reply:
Apparently the `'\''` escape works too, just like in bash
```
$ zsh
magnum@large ~ % echo 'node'\''s'
node's
```